tests/framework/microvm.py

```diff
@@ -207,9 +211,10 @@ def memory_events_queue(self, queue):
         """Set the memory usage events queue."""
         self._memory_events_queue = queue
 
-    def create_jailed_resource(self, path):
+    def create_jailed_resource(self, path, json_start=False):
```

Comment:
I find this counterintuitive, a helper function that hardlinks stuff shouldn't care about how Firecracker is started. I'd rename the parameter to `create_jail` (or something similar) instead.

Reply:
Renamed it.